packages/plugin-ext/src/plugin/tabs.ts

```diff
@@ -406,7 +406,7 @@ export class TabsExtImpl implements TabsExt {
         return this.proxy.$closeTab(extHostTabIds, preserveFocus);
     }
 
-    private async _closeGroups(groups: theia.TabGroup[], preserverFoucs?: boolean): Promise<boolean> {
+    private async _closeGroups(groups: theia.TabGroup[], preserverFocus?: boolean): Promise<boolean> {
```

Comment:
Looks like this is still not quite right:

```suggestion
    private async _closeGroups(groups: theia.TabGroup[], preserveFocus?: boolean): Promise<boolean> {
```